server.py

```diff
@@ -21,11 +21,19 @@
 
 
 async def background_tasks(app):
-    asyncio.create_task(application.start_sniffer_untrusted_agents())
-    asyncio.create_task(application.resume_operations())
-    asyncio.create_task(data_svc.load_data(directory='data'))
-    asyncio.create_task(data_svc.restore_state())
-    asyncio.create_task(application.run_scheduler())
+    try:
```

Comment:
we do the create_task in a number of places -- we should drop this on the base service class (or other) to do it in one spot.

Reply:
I changed it to an explicit `asyncio.get_event_loop()` which isn't deprecated and is backwards compatible.